bigquery/cloud-client/requirements.txt

```diff
@@ -1,5 +1,6 @@
 google-cloud-bigquery[pandas]==1.20.0
 google-auth-oauthlib==0.4.0
-ipython==7.8.0
+# ipython 5.9.0 is the last LTS release that supports python 2.7.
+ipython==5.9.0
 matplotlib==3.1.1
```

Comment:
looks like we'll have to do the same for matplotlib too - it'll be
matplotlib==2.2.5; python_version < "3.0"
matplotlib==3.1.1; python_version > "3.0"